_topic_maps/_topic_map_ms.yml

```diff
@@ -76,8 +76,12 @@ Name: Installing with RHEL image mode
 Dir: microshift_install_bootc
 Distros: microshift
 Topics:
-- Name: Installing with RHEL image mode
-  File: microshift-install-rhel-image-mode
+- Name: Understanding image mode for RHEL
+  File: microshift-about-rhel-image-mode
+- Name: Managing the bootc image
+  File: microshift-install-rhel-bootc-image
+- Name: Running the bootc image as a virtual machine
```

Comment:
I think we should try not to emphasise the virtual machine in the title. What about simply `Running the bootc image`?